lib/rules/no-unused-vars.js

```diff
@@ -624,6 +640,11 @@ module.exports = {
                             if (config.varsIgnorePattern && config.varsIgnorePattern.test(def.name.name)) {
                                 continue;
                             }
+
+                            // https://github.com/eslint/eslint/issues/15611
```

Comment:
It isn't necessary to refer to the issue for an enhancement like this one. It's clear that this code relates to the documented option. We're usually only adding links to issues for edge case bugs.